_posts/plater-blog-post.md

```diff
@@ -15,7 +15,7 @@ tags:
 - R
```

Comment:
@sckott can you please add these tags before merging?
- community
- software
- review